taxi/views.py

```diff
@@ -87,3 +102,19 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreateForm
+
```

Comment:
Consider adding a `success_url` attribute to `DriverCreateView` to specify the redirect URL after a successful form submission. For example, `success_url = reverse_lazy('taxi:driver-list')`.